crates/runtime-config/src/lib.rs

```diff
@@ -463,6 +469,63 @@ fn sqlite_config_resolver(
     ))
 }
 
+/// Resolves a runtime configuration for the variables factor from a TOML table.
```

Comment:
This code moving here was a necessary evil to decouple `spin-variables` from `spin-variables-vault` and `spin-variables-azure`